verticapy/toolbox.py

```diff
@@ -1610,3 +1610,300 @@ def __round__(self, x):
 
     def category(self):
         return self.category_
+
+#
+#
+# Tools to merge similar names/categories together.
+#
+# ---#
+def erase_prefix_in_name(name: str, prefix: list = []):
```

Comment:
Currently if we have two prefixes but the sequence inside the name does not match the sequence of prefix list then one of the prefix can remain. if that is intentional, then it is fine.

For example:
erase_prefix_in_name(name="pre1pre2waro",prefix=["pre2","pre1"])
OUPUT: 
'pre2waro'

UPDAtE: now that i re-read the details. i think you intend this to happen. SO PLEASE IGNORE IF THAT IS THE CASE